### PR TITLE
Cleaning Portal-api, remove Workflow and Data migration integration

### PR DESCRIPTION
**Pull request by IgnatG**
- remove Workflow Type-B schedulers
- remove all Workflow Type-A and Type-B functionality (including data migration functions)
- remove `useTfm` flag previously set on users/banks to conditionally submit to TFF instead of Workflow
- removed unused scripts in package.json
- updated packages & removed unused packages
- removed Sentry from portal-api
- removed Data Migration folder

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -156,15 +156,12 @@ services:
       - AZURE_PORTAL_STORAGE_ACCESS_KEY
       - AZURE_PORTAL_FILESHARE_NAME
       - AZURE_PORTAL_EXPORT_FOLDER
-      - DTFS_PORTAL_SCHEDULER
       - GOV_NOTIFY_API_KEY
       - GOV_NOTIFY_EMAIL_RECIPIENT
       - DTFS_PORTAL_SCHEDULER=true
-      - FETCH_WORKFLOW_TYPE_B_SCHEDULE=*/1 * * * * *
```
Comment (ttbarnes) [inline docker-compose.yml:163]: Can we add to the readme about this or add a comment to the referenced jira ticket? https://github.com/UK-Export-Finance/dtfs2#workflowtypeb-integration

Just incase we have to re-enable workflow

File: portal-api/src/scheduler/jobs/fetch-workflow-type-b.js [hunk 2]
```diff
@@ -1,154 +0,0 @@
-const moment = require('moment');
```
Comment (ttbarnes) [inline portal-api/src/scheduler/jobs/fetch-workflow-type-b.js:1]: RE my last comment sorry i've just seen this - can we update the jira ticket and link to this PR?

Comment (ttbarnes) [reply portal-api/src/scheduler/jobs/fetch-workflow-type-b.js:1]: Also we could now remove some workflow things in `deal-status.controller.js`

File: docker-compose.yml (hunk 1 above)
Comment (IgnatG) [reply docker-compose.yml:163]: good shout! - I'll do that

File: portal-api/src/scheduler/jobs/fetch-workflow-type-b.js (hunk 2 above)
Comment (IgnatG) [reply portal-api/src/scheduler/jobs/fetch-workflow-type-b.js:1]: we can indeed! I need to find out exactly where that is 😄

File: portal/server/graphql/queries/deals-query.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          firstname
+          surname
+        }
+        submissionDate
+        approvalDate
+        created
+      }
+    }
+  }
+}`;
```
Comment (ttbarnes) [inline portal/server/graphql/queries/deals-query.js:40]: is adding this query intentional?

**Review by ttbarnes — APPROVED** [review]
beautiful

File: portal/server/graphql/queries/deals-query.js (hunk 3 above)
Comment (IgnatG) [reply portal/server/graphql/queries/deals-query.js:40]: I thought you added this :D

Comment (ttbarnes) [reply portal/server/graphql/queries/deals-query.js:40]: 😂 OK either way we can remove it, it's now a simpler all-deals-query.js

Comment (ttbarnes) [reply portal/server/graphql/queries/deals-query.js:40]: Lots of PRs merged since this was opened 🙃

Comment (IgnatG) [reply portal/server/graphql/queries/deals-query.js:40]: I can take it out. there is one failing test that I need to fix anyway

Comment (IgnatG) [reply portal/server/graphql/queries/deals-query.js:40]: the beautiful part of this is that I can't see it in any of the commits :|

Comment (ttbarnes) [reply portal/server/graphql/queries/deals-query.js:40]: Amazing 😂

**Pull request merged**